resource/templates/key-value-list.html

```diff
@@ -0,0 +1,32 @@
+<div class="control-group">
+    <div class="key-value-list__labels flex">
+        <div class="key-value-list__labels_key">{{ keyLabel }}</div>
```

Comment:
Preferable to use `-` instead of `_`

```suggestion
        <div class="key-value-list__labels-key">{{ keyLabel }}</div>
```

Reply:
One day I'll know how to properly BEM 😁

Reply:
Actually I also have two classes, `.key-value-element__key` & `.key-value-element__value`. 

Maybe for consistency I should try to have all those classes look more alike?
Do you think it it better to use `.key-value-element__key`, `.key-value-element__value`, `.key-value-label__key` & `.key-value-label__value`?
Or `.key-value__label-key`, `.key-value__label-value`, `.key-value__element-key` & `.key-value__element-value`?

Reply:
I think it might be slightly cleaner to go with the first option.

Reply:
Was feeling the same way! Thanks :v:

Reply:
Done in fb235ed